mathutil_test.go

```diff
@@ -0,0 +1,107 @@
+package mathutil
```

Comment:
I didn't see any other tests in the project, so happy to omit this commit. But I wanted some sanity check that the functions were working properly.

Reply:
See [all_test.go](https://github.com/cznic/mathutil/blob/53c70789c7ff53526fd6c28c5a6881b138413e0c/all_test.go).

I think the new tests can be moved there, but it's not that much important.

Reply:
Will do!